src/components/ui/Heading/Heading.mdx

```diff
@@ -25,14 +25,14 @@ Heading rendered as a paragraph with color `blue`, `level = 3`.
 <ThemeProvider theme={lightTheme}>
   <GlobalStyles />
   <Canvas>
-    <Heading style={{ color: 'blue' }} level={3} tag="p">
+    <Heading style={{ color: 'blue' }} level={3} tag="span">
```

Comment:
This is to handle a non-breaking error in storybook that a `<p>` can not be nested within another `<p>`